testbed/testbed/results.go

```diff
@@ -61,8 +61,11 @@ func (r *PerformanceResults) Init(resultsDir string) {
 	r.perTestResults = []*PerformanceTestResult{}
 
 	// Create resultsSummary file
-	os.MkdirAll(resultsDir, os.FileMode(0755))
 	var err error
+	err = os.MkdirAll(resultsDir, os.FileMode(0755))
+	if err != nil {
+		log.Fatalf(err.Error())
```

Comment:
Just `log.Fatal(err)`? And everywhere.

Also to avoid var shading, better to write:
```
if err := os.MkdirAll(resultsDir, os.FileMode(0755)); err != nil {
	log.Fatal(err)
}
```

Reply:
Done